src/__test__/auth.test.js

```diff
@@ -29,10 +29,9 @@ describe('oauth', () => {
 
     // Await the request and get its reference.
     const request = await pendingRequest
-    const bodyParams = new URLSearchParams(request.body);
-    const [ headersSymbolName ] = Object.getOwnPropertySymbols(request.headers);
-    expect(request.url.href.includes('ims-na1.adobelogin.com/ims/token/v3')).toBeTruthy();
-    expect(request.headers[headersSymbolName]['content-type']).toEqual('application/x-www-form-urlencoded;charset=UTF-8');
+    const bodyParams = new URLSearchParams(await request.text());
+    expect(request.url.includes('ims-na1.adobelogin.com/ims/token/v3')).toBeTruthy();
+    expect(request.headers.get('content-type')).toEqual('application/x-www-form-urlencoded;charset=UTF-8');
```

Comment:
`request.headers` is now a plain `Headers` instance. You no longer need to meddle with internal symbols to check the request/response headers. Simple!